spec/index.bs

```diff
@@ -6649,8 +6691,26 @@ interface GPUQuerySet {
 GPUQuerySet includes GPUObjectBase;
 </script>
 
+{{GPUQuerySet}} has the following internal slots:
+
+<dl dfn-type=attribute dfn-for="GPUQuerySet">
+    : <dfn>\[[type]]</dfn> of type {{GPUQueryType}}.
+    ::
+        The type of queries managed by this {{GPUQuerySet}}.
+
+    : <dfn>\[[count]]</dfn> of type {{GPUSize32}}.
+    ::
+        The number of queries managed by this {{GPUQuerySet}}.
+
+    : <dfn>\[[pipelineStatistics]]</dfn> of type sequence<{{GPUPipelineStatisticName}}>.
+    ::
+        The set of {{GPUPipelineStatisticName}} values to determine which pipeline statistics will be returned in this {{GPUQuerySet}}.
```

Comment:
I know this is inconsistent with a lot of other places in the spec, but I think we should just store a single internal slot `[[descriptor]]` instead of splitting it out like this. We want to update the other places in the spec too, but haven't yet.

Reply:
If that, do we need to explain what these attributes mean in the descriptor interface?

Reply:
Yes, we would document them there. We have to document those anyway, so it avoids duplicating, or leaving one undocumented, or having to stub out one documentation with pointers to the other.

Reply:
Thanks, I've updated it.